Samples/Samples/View/CalendarPage.xaml

```diff
@@ -1,49 +1,61 @@
-                        <Label Text="{Binding Name}" TextColor="Black" HorizontalTextAlignment="Center"/>
-                    </StackLayout>
-                </ViewCell>
-            </DataTemplate>
-        </ResourceDictionary>
+             Title="Calendars">
+  <views:BasePage.Resources>
+      <ResourceDictionary>
+          <DataTemplate x:Key="EventItemTemplate">
+              <ViewCell>
+                  <StackLayout>
+                      <Label Text="{Binding Title}"
+                             TextColor="Black"
+                             HorizontalTextAlignment="Center"/>
+                  </StackLayout>
+              </ViewCell>
+          </DataTemplate>
+      </ResourceDictionary>
     </views:BasePage.Resources>
-    <Grid>
+  <views:BasePage.BindingContext>
+      <viewmodels:CalendarViewModel />
+    </views:BasePage.BindingContext>
+  <views:BasePage.Content>
+      <Grid>
         <Grid.RowDefinitions>
-            <RowDefinition Height="Auto"/>
-            <RowDefinition/>
+          <RowDefinition Height="Auto"/>
+          <RowDefinition Height="Auto"/>
+          <RowDefinition/>
         </Grid.RowDefinitions>
         <Grid>
-            <Grid.ColumnDefinitions>
-                <ColumnDefinition/>
-                <ColumnDefinition/>
-                <ColumnDefinition/>
-            </Grid.ColumnDefinitions>
-            <Button Text="Request Read Access"
-                    Command="{Binding RequestCalendarReadAccess}"
-                    HeightRequest="100"/>
-            <Button Grid.Row="0" Grid.Column="1" 
-                    Text="Request Write Access"
-                    Command="{Binding RequestCalendarWriteAccess}"/>
-            <Button Grid.Row="0" Grid.Column="2"  
-                    Text="Load Calendars" 
-                    Command="{Binding GetCalendars}"/>
+          <Grid.ColumnDefinitions>
+            <ColumnDefinition/>
+            <ColumnDefinition/>
+            <ColumnDefinition/>
+          </Grid.ColumnDefinitions>
+          <Button Text="Request Read Access"
+                  Command="{Binding RequestCalendarReadAccess}"
+                  HeightRequest="100"/>
+          <Button Grid.Row="0"
+                  Grid.Column="1"
+                  Text="Request Write Access"
+                  Command="{Binding RequestCalendarWriteAccess}"/>
+          <Button Grid.Row="0"
+                  Grid.Column="2"
+                  Text="Load Calendars"
+                  Command="{Binding GetCalendars}"/>
         </Grid>
-        <ListView Grid.Row="1" 
-                  ItemsSource="{Binding Calendars}"
-                  ItemTemplate="{StaticResource CalendarItemTemplate}"/>
-    </Grid>
+        <Picker Grid.Row="1"
+                Title="Select a Calendar"
+                ItemsSource="{Binding Calendars}"
+                ItemDisplayBinding="{Binding Name}"
+                SelectedItem="{Binding SelectedCalendar, Mode=TwoWay}"/>
+        <ListView Grid.Row="2"
```

Comment:
We should use CollectionView for this - hmmmm although I guess they may not be targetting XF4.3 yet. Can you confirm whether we can use CollectionView?

Reply:
I asked Scott to do that but they're not using 4.3 yet, therefore we went with ListView